RetailCoder.VBE/Refactorings/ExtractMethod/ExtractedMethod.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Rubberduck.Parsing.Grammar;
+using Rubberduck.Parsing.Symbols;
+
+namespace Rubberduck.Refactorings.ExtractMethod
+{
+    public class ExtractedMethod : IExtractedMethod
+    {
+        public string MethodName { get; set; }
+        public Accessibility Accessibility { get; set; }
+        public bool SetReturnValue { get; set; }
+        public ExtractedParameter ReturnValue { get; set; }
+        public IEnumerable<ExtractedParameter> Parameters { get; set; }
+        public string AsString()
```

Comment:
Because I think ToString() should provide JSON style of the content. AsString() is providing the ExtratedMethodSignature as a string.. So Perhaps better to call it CallSignature()?